### ISSUE TITLE
Expand search to include searching for issues

### ISSUE THREAD

**Issue by housseindjirdeh** [gitpoint/git-point#12]
Currently we can only search for users and repos. Would be nice to have the ability to [search for issues](https://developer.github.com/v3/search/#search-issues) as well.

**Comment by machour** [comment]
Should be doable right after #758 is merged

**Comment by housseindjirdeh** [comment]
@machour whewwheww I'm excited :) Benefits of making things more granular from the API level is already so easily noticeable
